rush-plugins/rush-resolver-cache-plugin/package.json

```diff
@@ -29,15 +30,21 @@
   },
   "exports": {
     ".": {
-      "require": "./lib/index.js",
-      "types": "./dist/rush-resolver-cache-plugin.d.ts"
+      "require": "./lib-commonjs/index.js",
```

Comment:
Order here is significant (the spec dictates that the first encountered condition in the object that matches any of the requested conditions is the result), so we should probably stick to a stable order across all of our package.json files.